code/Forms/FormMessageBootstrapAdapter.php

```diff
@@ -18,6 +18,7 @@ class FormMessageBootstrapExtension extends Extension
         'good' => 'alert-success',
         'bad' => 'alert-danger',
         'required' => 'alert-danger',
+        'validation' => 'alert-danger validation',
```

Comment:
- Keeping `validation` for projects that might be using that css class as a selector for changed styling or for js logic.
- Need `alert-danger` for detecting the error in https://github.com/silverstripe/silverstripe-admin/blob/16f8e525c3ac140fc718cd89a7c8cd3cc065ac71/client/src/legacy/LeftAndMain.EditForm.js#L154-L159